specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Dataset.json

```diff
@@ -23,6 +23,10 @@
           "type": "object",
           "description": "Columns that define the structure of the dataset. Type: array (or Expression with resultType array), itemType: DatasetDataElement."
         },
+        "schema": {
```

Comment:
Was this added newly to this version? Or was it there from the beginning but failed to be added in the swagger?

Reply:
It was newly added.